### ISSUE TITLE
Email notifications not working

### ISSUE THREAD

**Issue by netnazgul** [online-go/online-go.com#58]
...since migrating onto new version. Mentioned here: [https://forums.online-go.com/t/no-more-email-notification-since-upgrade/10569/22](url)

On a side note I'd like to ask an option for "your move" notifications to be sent only once there is a game to play and do not send email on EVERY game that becomes available. But probably this should go to another issue.

**Comment by netnazgul** [comment]
damn, only posted this and THEN checked that the issue was closed some hours ago ;)
